cypress/integration/cypressActions.spec.js

```diff
@@ -1,14 +1,51 @@
 /// <reference types='cypress' />
 
 // Basic level:
-// 1. Fill all fields in forms except "picture" 
+// 1. Fill all fields in forms except "picture"
 // 2. Click on [Submit] button
 // 3. Validate inputed data in modal window
 // Site: https://demoqa.com/automation-practice-form
 
+let user;
+
+describe('Your third Test', () => {
+    before(function () {
+        cy.task("freshUser").then((object) => {
+            user = object;
+        });
+    });
+
+    // Registration
+    it('Registration', () => {
+        cy.visit('https://demoqa.com/automation-practice-form');
+        cy.get('[placeholder="First Name"]').type(user.username);
+        cy.get('[placeholder="Last Name"]').type(user.username2);
+        cy.get('#userEmail').type(user.email);
+        cy.get('#gender-radio-1').click({force: true});
+        cy.get('[placeholder="Mobile Number"]').type('0102030405');
+        cy.get('#dateOfBirthInput').type('{selectAll}').type('10 June 2000{enter}');
+        cy.get('#subjectsContainer').type('Ch{downArrow}{enter}');
+        cy.get('#hobbies-checkbox-1', {timeout:2000}).click({force: true});
+        cy.get('[placeholder="Current Address"]').type(user.address);
+        cy.contains('.css-1hwfws3', 'Select State').click().type('Haryana{enter}');
+        cy.get('#city').click().type('Karnal{enter}');
+        cy.get('#submit').click();
+        cy.contains('tbody tr', 'Student Name').should('contain', user.username + ' ' + user.username2);
```

Comment:
Better divide it into two different sections (adding a student, validating data in the table)